CONTRIBUTING.md

```diff
@@ -1,55 +0,0 @@
-# How to Contribute
```

Comment:
There is no need to delete this file

Reply:
It was aligned with other extension libraries, so it was removed.